numpy/ma/tests/test_core.py

```diff
@@ -1990,6 +1991,177 @@ def test_datafriendly_mul_arrays(self):
         assert_equal(a, [[1, 1], [3, 3]])
         assert_equal(a.mask, [[0, 1], [0, 1]])
 
+    def test_inplace_addition_scalar_uint8(self):
+        with warnings.catch_warnings(record=True) as w:
+            # Test of inplace additions
```

Comment:
Usually `catch_warnings` is followed by something like
```
            warnings.filterwarnings('always', '', UserWarning)
```
Although in these cases you will probably see them all as they are in different places.